### ISSUE TITLE
Nice to have: options ordered from best to worst

### ISSUE THREAD

**Issue by ThomasCabaret** [lightvector/leela-analysis#11]
Hi, we can have this kind of display (see below).
It would be good to have it sorted and labelled accordingly.

A -> Win%: 49.01% (8200 visits) 
B -> Win%: 47.03% (12611 visits) 
C -> Win%: 46.78% (5269 visits) 
D -> Win%: 47.38% (4509 visits) 
E -> Win%: 46.98% (4082 visits) 
F -> Win%: 44.97% (1787 visits) 
G -> Win%: 46.81% (736 visits)

**Comment by lightvector** [comment]
No, I think changing the sorting is not a good idea. This is the order that
Leela returns the moves, which as far as I can tell is the order that Leela
actually prefers the moves in. If I recall correctly from watching the
analysis window during interactive with Leela in the GUI version, Leela
neither strictly uses highest visit count (which is influenced by the
policy net) nor uses the highest winrate (which comes from the value net
and playouts) when choosing a move, but rather uses a combination of them.
So maybe it looks unsorted but this is actually Leela's order of preference
of all the moves.
